ios/RNModules/RNLogging/RNLogging/RNLoggingModule.h

```diff
@@ -2,8 +2,8 @@
 
 @protocol RNLogger <NSObject>
 
-- (void)ancillaryDisplayed:(nullable NSString *)type;
-- (void)ancillaryPurchased:(nullable NSString *)type;
+- (void)ancillaryDisplayed:(nullable NSString *)type andProvider:(nullable NSString *)provider;;
```

Comment:
Please, do not use `and` in name of the last parameter. Although, "and" sounds good in this case, it makes addition of next parameter harder. 

More info about naming: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html 

> Don’t use “and” to link keywords that are attributes of the receiver. Although “and” may sound good in this example, it causes problems as you create methods with more and more keywords.

Reply:
Thanks for that. I've changed it to just `provider`.